### PR TITLE
disable rainbow

### LINKED ISSUE #98: Investigate OpenSSH slowdown due to Rainbow
When #97 has landed, it would be goodness to find out why everything slows down by at least an order of magnitude if a Rainbow L3 algorithm is activated.

### PR DESCRIPTION
**Pull request by baentsch**
Fixes #98 . OQS-v8 doesn't have Rainbow active by default any more.

### CODE REVIEW

File: ssh-keygen.c [hunk 1]
```diff
@@ -241,10 +241,8 @@ type_bits_valid(int type, const char *name, u_int32_t *bitsp)
 	if (IS_ECDSA_HYBRID(type)) {
 		switch (type) {
 ///// OQS_TEMPLATE_FRAGMENT_SET_BITS_START
```
Comment (christianpaquin) [inline ssh-keygen.c:241]: Do we need this switch statement now, since it looks like all cases were for rainbow?

**Review by christianpaquin — COMMENTED** [review]
Looks good. One question in `ssh-keygen.c` wrt a fragment that now looks empty without rainbow.

File: ssh-keygen.c (hunk 1 above)
Comment (baentsch) [reply ssh-keygen.c:241]: This code is caused by an error in the oqs-template jinja script. Given this is for code we're effectively not maintaining any more, let me see whether it's simple to change...

Comment (baentsch) [reply ssh-keygen.c:241]: It was. See new commit.

**Review by christianpaquin — APPROVED** [review]

**Pull request merged**